sdks/python/apache_beam/io/hadoopfilesystem_test.py

```diff
@@ -538,6 +539,14 @@ def test_checksum(self):
     self.assertEqual(
         'fake_algo-5-checksum_byte_sequence', self.fs.checksum(url))
 
+  def test_last_updated(self):
+    url = self.fs.join(self.tmpdir, 'f1')
+    with self.fs.create(url) as f:
+      f.write(b'Hello')
+    tolerance = 60  # 1 min
+    result = self.fs.last_updated(url)
+    self.assertAlmostEqual(result, time.time(), delta=tolerance)
```

Comment:
1 minute is a reasonable window, but this test looks like it could be flaky still. Is there a way to make this deterministic?

Reply:
This mirrored an once sickbayed test (s3io_test.test_last_updated)[https://github.com/apache/beam/blob/da6acf212e93aef266630f36624f5d23a1a93801/sdks/python/apache_beam/io/aws/s3io_test.py#L116] which set tolerance to 5 min. I did not the reasoning of this original setting, but once the failure has resolved, the time difference should be instant: it is just the interval between fake file creation and the assertion statement. If it generates confusion we could set the interval to be the same small value in both place.

Reply:
I know s3 has eventual consistency, so that may be the reason why? (though 5minutes still sounds extremely high)